identity/cache.go

```diff
@@ -1,15 +1,14 @@
 package identity
 
 import (
-	"github.com/mysterium/node/service_discovery/dto"
 	"io/ioutil"
 	"encoding/json"
 	"os"
 	"path/filepath"
 )
 
 type cacheData struct {
-	Identity dto.Identity `json:"identity"`
+	Identity Identity `json:"identity"`
```

Comment:
should not be string?

Reply:
Doesn't matter. We can store the full structure in the cache file.